core/rpc/src/system/mod.rs

```diff
@@ -60,6 +60,13 @@ pub trait SystemApi<Hash, Number> {
 	/// Returns currently connected peers
 	#[rpc(name = "system_peers")]
 	fn system_peers(&self) -> Result<Vec<PeerInfo<Hash, Number>>>;
+
+	/// Returns current state of the network.
+	///
+	/// **Warning**: This API is not stable.
+	// TODO: make this stable and move structs https://github.com/paritytech/substrate/issues/1890
+	#[rpc(name = "system_network_state")]
```

Comment:
Not 100% sure why this is not `system_networkState`, i.e. we use camelCase everywhere, see e.g. https://github.com/paritytech/substrate/blob/master/core/rpc/src/author/mod.rs#L63

Reply:
Yeah, the convention in general is `<namespace>_<camelCasedMethodName>`.

Reply:
Maybe let's use `experimental_` or `unstable_` prefix to indicate that the API is not going to last?

Reply:
> Maybe let's use `experimental_` or `unstable_` prefix to indicate that the API is not going to last?

What's the general opinion on that?

Reply:
I'm guessing it is going to last, however we are not 100% sure about the structs yet. (As an initial effort, just treating these as a blob, i.e. there are no specific implemented structures to make sense of what is provided - whatever is retrieved will be dumped as-is)

With aliases in mind (once it swaps to stable, there is an alias to the original version), a different prefix won't hurt either way since "old" calls will map to the "new, stable", so have no strong opinions as to "place it at final position" or "really indicate as experimental".